### ISSUE TITLE
Unable to copy or move Session object

### ISSUE THREAD

**Issue by EMHussain** [whoshuu/cpr#553]
compile error on following statement
```
 std::vector<cpr::Session> sessionArray(10);
 sessionArray.push_back(cpr::Session());
```
error: invalid application of 'sizeof' to an incomplete type 'cpr::Session::Impl

**Comment by EMHussain** [comment]
I am using version 1.6

**Comment by COM8** [comment]
Ok. I will have a look at it. might be a regression bug.
Thanks for reporting.

**Comment by jsouquie** [comment]
Hi, this bug was fixed in cpr v1.6.1, see #544

**Comment by COM8** [comment]
Thought so too.

Closing this for inactivity.
Feel free to reopen in case this issue still persists.
